wiki/Tournaments/GBC/GBC_2023_Spring/en.md

```diff
@@ -0,0 +1,207 @@
+| Statistician & wiki editor | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308), ::{ flag=CN }:: [FAIRYBYUN](https://osu.ppy.sh/users/17575228) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1730051)
+- Livestream
+  - [Bilibili stream](https://live.bilibili.com/22545296) (::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407))
+  - [Bilibili stream](https://live.bilibili.com/10183) (::{ flag=CN }:: [aiyulu](https://osu.ppy.sh/users/189617))
+  - [Bilibili stream](https://live.bilibili.com/6605693) (::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273))
+- [GBC QQ Group](https://jq.qq.com/?_wv=1027&k=zqBeTqMW)
+- [Information spreadsheet](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY?tab=uitssi)
+
+## Participants
+
+| Group | Player(s) |
+| :-: | :-- |
+| Group EX | *TBD* |
+| Group S | *TBD* |
+| Group A | *TBD* |
+| Group B | *TBD* |
+
+## Match results
+
+Due to the large number of players, the match results will be displayed in the [information spreadsheet](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY?tab=uitssi).
+
+## Ruleset
+
+### General
+
+1. The GB Cup 2023 Spring is a point-based weekly group tournament, played on the osu!mania game mode.
+   - Points earned by players in weekly matches are converted based on the total ranking of the mappool.
+   - The total ranking of the mappool is based on the sum of the rankings of each player in each map.
+2. Beatmap scoring is based on Score V2.
+3. The GB Cup 2023 Spring consists of **4 groups (EX, S, A, and B)**, with only **2 groups** playing each week.
+   - Each group has an overrun buffer of *100 pp*.
+   - If a player upgrades after the start of the tournament, 50% of the points will be kept.
+4. The maps for each round will be announced by the mappool selectors on the official stream, on the Sunday before the actual matches take place.
+   - Each mappool contains 7 beatmaps of varying types.
+5. Players schedule their match time before the start of the week's match.
+   - Players with special time requirements need to contact the referee in advance. The organiser does not guarantee that anyone's request will be met, and players need to properly communicate with the referee in advance.
```

Comment:
duplicate of scheduling information point 3

that one mentions to contact the organiser instead of a referee. which is it?

Reply:
Replaced 'referee' with 'Organizer'